src/driver/sap/SapDriver.ts

```diff
@@ -246,55 +252,37 @@ export class SapDriver implements Driver {
     async connect(): Promise<void> {
         // HANA connection info
         const dbParams = {
-            hostName: this.options.host,
-            port: this.options.port,
-            userName: this.options.username,
-            password: this.options.password,
+            serverNode: `${this.options.host}:${this.options.port}`,
+            uid: this.options.username,
```

Comment:
`uid` and `pwd` are the old options, `user` and `password` are preferred nowadays since they are not abbreviated.